server/php-master.cpp

```diff
@@ -825,6 +834,7 @@ int run_worker() {
   worker->logname_id = worker_logname_id;
   worker->last_activity_time = my_now;
 
+  worker->task_result_fd_idx = task_result_fd_idx;
```

Comment:
we already have `logname_id` and I guess it does exactly the same stuff as `task_result_fd_idx`. I believe, that we can reuse it somehow, maybe it's worth to rename `logname_id` for better understanding.